StortSpelProjekt/Engine/src/Renderer/QuadManager.h

```diff
@@ -0,0 +1,37 @@
+#ifndef QUADMANAGER_H
+#define QUADMANAGER_H
+
+#include <map>
+#include <vector>
+
+class Texture;
+class Mesh;
+class Resource;
+class ShaderResourceView;
+class Eventbus;
+
+struct Vertex;
+struct MouseClick;
+
+class QuadManager
+{
+	public:
+		QuadManager();
+		~QuadManager();
+
+		void CreateQuad(float2 pos, float2 size, bool clickable, std::wstring texturePath = L"NONE");
```

Comment:
I know what you mean with texturePath.. but in this case, since it wont be a path, isn't it better to just have an id like: quad0, quad1, quad2 etc.. which a static unsigned int which you can increment in this class and give each quad a unique id? And also while you're at it, you might aswell add a "operator==", if we would need to compare two quads in the future.

Reply:
It is a texture path?

Reply:
okay done